content/docs/tutorials/data-exploration-in-r/intro-to-r.Rmd

```diff
@@ -170,6 +170,10 @@ head(mobility)
 ncol(mobility)
 ```
 
+```{r, echo = TRUE, webex.hide = 'Optional solution', results = 'hide', warning = FALSE}
+# optional solution 
+mobility[c(1,5,6,7)]<-list(NULL)
```

Comment:
the problem w/ this solution is that it may break - suppose the columns change, then column 5 is not what is intended here. So - I would rather classify this is a very suboptimal solution. Maybe you point it out in the code here? It's useful you did it, sure, but it is not a good solution.